patch/cisco-Enable-static-memory-reservation-for-OIRable-PCIe-de.patch

```diff
@@ -0,0 +1,198 @@
+commit 9a57f4339767fb8129ee54e9d1714af59d4a6937
+Author: balsup <balsup@contoso.com>
+Date:   Fri Aug 14 00:09:18 2020 +0000
+
+    Reserve address space for powered-off devices behind bridges
+
+diff --git a/drivers/pci/setup-bus.c b/drivers/pci/setup-bus.c
+index 8e5b00a..062cbe1 100644
+--- a/drivers/pci/setup-bus.c
++++ b/drivers/pci/setup-bus.c
+@@ -979,6 +979,149 @@ static inline resource_size_t calculate_mem_align(resource_size_t *aligns,
+ 	return min_align;
+ }
+ 
++#define PLX_RES_MAGIC_VALUE            0xABBA
++#define PLX_RES_DS_PORT_REG0           0xC6C
++#define PLX_RES_DS_PORT_REG1           0xC70
++#define PLX_RES_MAGIC_OFFSET           0xC76
++#define PLX_RES_NP_MASK                0x1
++#define PLX_RES_P_MASK                 0x1F
++
++static struct pci_dev *
++plx_find_nt_device(struct pci_bus *bus, unsigned short brg_dev_id)
++{
++       struct pci_dev *dev, *nt_virt_dev = NULL;
```

Comment:
There are again several style issues? Please check with `checkpatch.pl` next time.

Reply:
Fixed coding style issues and verified running checkpatch.pl